ktor-shared/build.gradle.kts

```diff
@@ -5,6 +5,7 @@ subprojects {
         val commonMain by getting {
             dependencies {
                 api(project(":ktor-http"))
+                api(project(":ktor-http:ktor-http-cio"))
```

Comment:
Is it required here or can it be moved to a nested modules?

Reply:
Moved to `ktor-share:ktor-serialization`